src/System.Management.Automation/engine/MshMemberInfo.cs

```diff
@@ -4114,16 +4125,15 @@ public override void Remove(string name)
                     name);
             }
 
-            lock (_members)
+            lock (Members)
```

Comment:
Shall we also check for `_members == null` for the `Remove` action as a shortcut path?

Reply:
I think we can add the check in public methods.

Fixed.